### DISCUSSION TITLE
Missing Image Pinch Button After Updating Uploader to Latest Version

### DISCUSSION THREAD

**Discussion by egoing** [uploadcare/file-uploader#686] [blocks] [answered]
After updating the Uploader to the latest version, the image pinch button is no longer visible. Has this issue been reported before?

<img width="537" alt="image" src="https://github.com/uploadcare/blocks/assets/303822/00dbdd4c-470f-4695-8011-7b589f305f08">

**Comment by egordidenko** [comment, marked as answer]
Hi, @egoing!

Thank you for the appeal.
Added an icon for editing and made a new release. Please try to use the current version [0.46.3](https://www.npmjs.com/package/@uploadcare/blocks/v/0.46.3).
